apps/www/server/index.js

```diff
@@ -47,6 +48,34 @@ app.prepare().then(() => {
     }),
   )
 
+  if (process.env.DENY_IPS) {
+    try {
+      // We use JSON.parse because to filter a IP range it needs to be specified as an array, e.g. [['127.0.0.1', '127.0.0.10']]
+      const denyIPs = process.env.DENY_IPS
+        ? JSON.parse(process.env.DENY_IPS)
+        : []
```

Comment:
If there is no IP to deny requests for, setting up `server.use(ipfilter(...))` is probably not necessary?

Reply:
Ah yes, actually the whole check was unnecessary because there's an `if` clause before.